libs/vkd3d/resource.c

```diff
@@ -1155,6 +1155,112 @@ HRESULT vkd3d_get_image_allocation_info(struct d3d12_device *device,
     return hr;
 }
 
+static void d3d12_resource_get_tiling(struct d3d12_device *device, struct d3d12_resource *resource,
+        UINT *total_tile_count, D3D12_PACKED_MIP_INFO *packed_mip_info, D3D12_TILE_SHAPE *tile_shape,
+        D3D12_SUBRESOURCE_TILING *tilings, VkSparseImageMemoryRequirements *vk_info)
+{
+    const struct vkd3d_vk_device_procs *vk_procs = &device->vk_procs;
+    VkSparseImageMemoryRequirements *memory_requirements = NULL;
+    const D3D12_RESOURCE_DESC *desc = &resource->desc;
+    unsigned int i, tile_count, packed_tiles;
+    uint32_t memory_requirement_count = 0;
+    VkExtent3D block_extent;
+
+    memset(vk_info, 0, sizeof(*vk_info));
+
+    if (desc->Dimension == D3D12_RESOURCE_DIMENSION_BUFFER)
+    {
+        tile_count = align(desc->Width, VKD3D_TILE_SIZE) / 65536;
```

Comment:
Why state VKD3D_TILE_SIZE and 64k separately?

Reply:
Fixed.